docs/guide/php-attributes.md

```diff
@@ -0,0 +1,234 @@
+#### Using attributes
+
+##### Basic usage
```

Comment:
```suggestion
##### Basic usage

Usage of [PHP attributes](https://www.php.net/manual/en/language.attributes.overview.php) to define validation rules provides extra reading convenience since they are residing close to fields they are defining.
```